### ISSUE TITLE
Header Error

### ISSUE THREAD

**Issue by piyusharorawork** [cthackers/adm-zip#269]
while extracting the zip file using 7zip , I am getting the header error . 
Please help in this regard

![image](https://user-images.githubusercontent.com/31386968/49635831-f1f70880-fa26-11e8-93fa-f80adc94ff2d.png)

**Comment by remidenele** [comment]
I experienced the same issue when trying to unzip, with 7-Zip (tested with latest 18.06), any archive created with adm-zip (tested with 0.4.13) in a Nodejs 8 env. All files are successfully extracted from the archive but an `Headers error` is always displayed:

![7-zip-headers-error](https://user-images.githubusercontent.com/8693949/50596208-c17a8100-0ea3-11e9-8178-95ad23de8968.png)

A simple reproduction code:

```javascript
import * as AdmZip from 'adm-zip';

const zipArchive = new AdmZip();
zipArchive.addFile('test-file.txt', Buffer.from('Simple text file'));
zipArchive.writeZip('./test-file.zip');
```

I tried to test the integrity of the archive with the 7-Zip CLI and just got `Unsupported method`:

![7zip-command-line-test](https://user-images.githubusercontent.com/8693949/50596383-65fcc300-0ea4-11e9-9ba0-9729be4cda2b.png)

Here is the hex value of the zip archive:

![test-file zip-hex](https://user-images.githubusercontent.com/8693949/50596472-b542f380-0ea4-11e9-8a18-cd1b0b1b18f9.png)

I don't know how to go deeper in the debug. I will try to post to the 7-Zip forum too.

**Comment by remidenele** [comment]
[Answer](https://sourceforge.net/p/sevenzip/discussion/45797/thread/2a853dcff0/#6082) from Igor Pavlov, developer of 7-Zip, regarding the zip archive generated with the repro code in my previous comment is: 

> Zip archive contains two copies of file metadata (local and central).
> And these copies have different CRC32 values for file

**Comment by jcowgar** [comment]
I use adm-zip to read/write .docx files (which are just .zip files). Microsoft Word states that the .docx file is corrupt. It repairs it fine, but not without the annoying dialog when opening. I can confirm that PR #267 makes Microsoft Word read the .docx file w/o this issue.

**Comment by zakius** [comment]
While this PR got merged into master already it hasn't been updated at npm yet, is there any planned date of release?

**Comment by fpsqdb** [comment]
When will the new version come out?

**Comment by h3llrais3r** [comment]
@cthackers, Any plans for a release with this fix? Thx!

**Comment by rcrvano** [comment]
I Have this error too

**Comment by jonathanstokes** [comment]
> I use adm-zip to read/write .docx files (which are just .zip files). Microsoft Word states that the .docx file is corrupt. It repairs it fine, but not without the annoying dialog when opening. I can confirm that PR #267 makes Microsoft Word read the .docx file w/o this issue.

I have this same `.docx` problem, and in case this helps someone else: I see the problem with adm-zip `0.4.13`, but I don't see it with `0.4.11`.

**Comment by roeyb** [comment]
this seems to have been fixed as of version 0.4.14
